### DISCUSSION TITLE
Notification area improvements

### DISCUSSION THREAD

**Discussion by gaabster** [PranshulGG/WeatherMaster#147] [Ideas]
Hello.

Is it possible to have current temperature in the notification? At the moment we have **max/min**.
Another thing would be to have the notification refresh every **x hours** or have a **refresh** button when you expand the notification. I don't even know if this is possible.

Thank you for this wonderful app.

**Comment by PranshulGG** [comment]
> Hello.
> 
> Is it possible to have current temperature in the notification? At the moment we have **max/min**. Another thing would be to have the notification refresh every **x hours** or have a **refresh** button when you expand the notification. I don't even know if this is possible.
> 
> Thank you for this wonderful app.

Notifications still need some work done!
